assets/memberships-gate/editor.js

```diff
@@ -63,6 +83,27 @@ const GateEdit = ( { editPost, createNotice, meta } ) => {
 						) }
 					/>
 				) }
+				{ meta.style === 'overlay' && (
+					<Fragment>
+						<SelectControl
+							label={ __( 'Size', 'newspack' ) }
+							value={ meta.overlay_size }
+							options={ overlaySizes }
+							onChange={ value => editPost( { meta: { overlay_size: value } } ) }
+						/>
+						<PositionControl
+							label={ __( 'Position', 'newspack' ) }
+							value={ meta.overlay_position }
+							allowedPositions={ [ 'bottom', 'center' ] }
```

Comment:
Perhaps because I was spreading it from `props` down to the `ButtonGroup` by accident? Couldn't reproduce locally but I'm no longer spreading it down. Please let me know if the error is gone

Reply:
Yes, no longer seeing it!